c++/src/kj/compat/url.h

```diff
@@ -43,6 +43,14 @@ struct UrlOptions {
   // silently removed. In other words, setting this false causes consecutive slashes in the path or
   // consecutive ampersands in the query to be collapsed into one, whereas if true then they
   // produce empty components.
+
+  enum class MergeFragment {
+    kAuto, // Depend on the context if we merge the fragment into the previous component.
```

Comment:
We name enumerants (and constants in general) using UPPER_SNAKE_CASE. I know there are a lot of problems with this naming style, but it is what we use and we should stay consistent.

Reply:
That's fine by me! I think the primary argument I've seen is macro collision. I'd rather people just use less macros when they can.